test/deploy-prepare-and-test-pipeline.sh

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+PRJ_ROOT="$(cd `dirname "${BASH_SOURCE}"`/..; pwd)"
+
+ENV_CODE=${1:-${ENV_CODE}}
+
+BATCH_ACCOUNT_NAME=${2:-${BATCH_ACCOUNT_NAME:-"stellarbatchdev"}}
```

Comment:
Since we are making this repo public, can we remove references to our internal batch accounts?

Reply:
It may have been convenient during the testing. But yes we should remove these defaults and also if there any references to internal sub, etc that should also be removed

Reply:
removed